ml-pipeline/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Document classification",
```

Comment:
Update metadata like title, short_description and tags, because this tutorial differs from the "Document classification" tutorial